packages/react/src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,106 @@
+import cl from 'clsx/lite';
+import { type HTMLAttributes, type ReactNode, forwardRef } from 'react';
+import { Paragraph, type ParagraphProps } from '../Typography';
+
+export type BadgeProps = {
+  /**
+   * The color of the badge
+   *
+   * @default accent
+   */
+  color?: 'accent' | 'info' | 'success' | 'warning' | 'danger' | 'neutral';
+  /**
+   * The size of the badge
+   *
+   * @default md
+   */
+  size?: 'sm' | 'md' | 'lg';
+  /**
+   * The number to display in the badge
+   */
+  count?: number;
+  /**
+   * The maximum number to display in the badge, when the count exceeds this number, the badge will display `{max}+`
+   */
+  maxCount?: number;
+  /**
+   * The placement of the badge
+   *
+   * @default top-right
+   */
+  placement?: 'top-right' | 'top-left' | 'bottom-right' | 'bottom-left';
```

Comment:
Can we adapt this to use `import type { Placement } from '@floating-ui/react';` ? 

Would be nice to have consistent values under `placement` props for our components. 

As of now `Tooltip` is the only one which defeers from other placement values.

Reply:
This does not use floating-ui, and the types of `placement` we use in the two other components does not match how the `Badge` is placed. 

`Popover` has three placements per side, `-start`, `middle` and `-end`, none of these intersect with `Badge`.
`Tooltip` only has one placement per side, which also does not intersect with `Badge`, because it is not in any of the corners.

I suggest we don't use the type, because the placement is different for all three components.

Reply:
Aight, fair enough, but no more than 3 different placement variations then! 😅

Reply:
I think we have managed to cosume all variations of placement in these three components 😅